src/db/options.rs

```diff
@@ -39,25 +39,25 @@ impl From<CreateCollectionOptions> for Document {
             document.insert("max", Bson::I64(max));
         }
 
-        let mut flags = 0;
+        let mut flags = 0_i32;
```

Comment:
Is there a reason that the type is needed here? (If it is, I think I'd prefer `let mut flags: i32 = 0;`)

Reply:
I was not sure how strict the consumer of the returned document will be about types (I have been bitten in the past by supplying an `I32` where an `I64` was expected…), so I wanted to be explicit. Anyway, the type of integer literals defaults to `i32` so we can technically get rid of the type annotation.